test/alternator/test_encoding.py

```
@@ -309,3 +312,61 @@ def test_index_key_not_schema_column(dynamodb, cql, table_with_indexes):
     assert attrs['y'] == b'\x00' + b'world'
+
+# Pre-create with CQL the keyspace "alternator_{name}" which Alternator uses
+# for a table named {name}, with tablets enabled or disabled - so that a table
+# created afterwards has a pre-configured keyspace to re-use.
+@contextmanager
+def precreated_keyspace(cql, this_dc, name, tablets):
+    try:
+        cql.execute(f'CREATE KEYSPACE "alternator_{name}" WITH REPLICATION = '
+            f"{{'class': 'NetworkTopologyStrategy', '{this_dc}': 1}} "
+            f"AND TABLETS = {{'enabled': {str(tablets).lower()}}}")
+    except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)